packages/fhir-team-management/src/components/TeamsAddEdit/Form.tsx

```diff
@@ -0,0 +1,230 @@
+import React, { useState } from 'react';
+import { Select, Button, Form as AntdForm, Radio, Input } from 'antd';
+import { history } from '@onaio/connected-reducer-registry';
+import { v4 } from 'uuid';
+import {
+  TEAMS_GET,
+  PRACTITIONERROLE_DEL,
+  PRACTITIONERROLE_GET,
+  PRACTITIONER_GET,
+} from '../../constants';
+import {
+  sendSuccessNotification,
+  sendInfoNotification,
+  sendErrorNotification,
+} from '@opensrp/notifications';
+import { Organization, OrganizationDetail, Practitioner, PractitionerRole } from '../../types';
+import { useQueryClient } from 'react-query';
+
+import lang from '../../lang';
+import FHIR from 'fhirclient';
+
+const layout = { labelCol: { span: 8 }, wrapperCol: { span: 11 } };
+const offsetLayout = { wrapperCol: { offset: 8, span: 11 } };
+
+export interface FormField {
+  team?: OrganizationDetail;
+  name: string;
+  active: boolean;
+  practitioners: string[];
+}
+
+interface Props {
+  fhirbaseURL: string;
+  allPractitioner: Practitioner[];
+  allPractitionerRole?: PractitionerRole[];
+  initialValue?: FormField;
+}
+
+/**
+ * Handle form submission
+ *
+ * @param {string} fhirbaseURL - base url
+ * @param {object} initialValue initialValue of form fields
+ * @param {object} values value of form fields
+ * @param {Practitioner} practitioner list of practitioner to refer to when adding or removing
+ * @param {PractitionerRole} allPractitionerRole list of practitionerRole to remove or add
+ */
+export async function onSubmit(
+  fhirbaseURL: string,
+  initialValue: FormField,
+  values: FormField,
+  practitioner: Practitioner[],
+  allPractitionerRole?: PractitionerRole[]
+) {
+  const officialidentifier = initialValue.team
+    ? initialValue.team.identifier?.find((e) => e.use === 'official')?.value
+    : v4();
+
+  const payload: Organization = {
+    resourceType: 'Organization',
```

Comment:
i have implemented intellisense so if user enters wrong info it will show error 

![image](https://user-images.githubusercontent.com/19289015/132404111-5375b3d1-1303-4756-bde8-93beee9254ed.png)